src/panels/lovelace/hui-editor.ts

```diff
@@ -96,6 +96,7 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
         }
 
         textarea {
+          box-sizing: border-box;
           height: calc(100% - 16px);
```

Comment:
Btw where does this 16px come from? I have white space underneath it now, but on 100% it is to big and I get a scrollbar... Here -4px seems to be the sweet spot but don't get where the 4pxs come from...

Reply:
Not that it is really important 😃

Reply:
Oh uhmm yeah I don't know

Reply:
Oh, 16px is because of the padding with default box model. With border-box, you will have to include padding as part of the height I think?

Reply:
`
The width and height properties (and min/max properties) includes content, padding and border
`

Reply:
Mmm ok, that's why it was showing me scrollbars before I made it border-box, so now it SHOULD just be `height: 100%` still don't know where the 4px come from... 🤔